.changeset/loud-shoes-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": minor
+---
+
+Applied LimitMultiplier to chainSpecificGasLimit in all gas estimators. Updated TestArbitrumEstimator and TestSuggestedPriceEstimator to reflect this addition.
```

Comment:
nit: Don't think we've really standardized this and I've seen other teams do it but we probably shouldn't mention anything about tests in changesets. I think we want to keep the changelog message just to things that are externally relevant. But maybe the team can put in their opinions here too if they disagree

Reply:
That makes sense to me. Changes to test should be implied